clab/config/utils.go

```diff
@@ -168,25 +171,25 @@ func linkIP(link *types.Link) (string, string, error) {
 	_, okA := link.A.Node.Config.Vars[vkSystemIP]
 	_, okB := link.B.Node.Config.Vars[vkSystemIP]
 	if okA != okB {
-		return "", "", fmt.Errorf("%s var required on all nodes", vkSystemIP)
+		log.Warnf("to auto-generate link IPs, a %s var is required on all nodes", vkSystemIP)
```

Comment:
let's expand var to variable? since this is a user-facing error, I think it is better to spell the var as a variable